src/compas_ags/utilities/equilibrium.py

```diff
@@ -43,6 +50,80 @@ def check_deviations(form, force, tol=10e-3):
         else:
             checked = False
             a = min(a, 180 - a)
+        if a > a_max:
+            a_max = a
         form.edge_attribute(edges_form[i], 'a', a)
 
+    if printout:
+        if a_max > tol:
+            print('> Equilibrium Check | Max deviation:', a_max)
+
+    return checked
+
+
+def check_force_length_constraints(force, tol=10e-3, printout=False):
```

Comment:
`10e-3 == 0.01 == 1e-2`
therefore the second formulation is clearer i find

Reply:
ok, modified